stable/kommander/charts/kommander-thanos/values.yaml

```diff
@@ -45,10 +47,18 @@ thanos:
     - "--grpc-client-tls-ca=/etc/certs/ca.crt"
     - "--grpc-client-server-name=server.thanos.localhost.localdomain"
     certSecretName: kommander-thanos-client-tls
+    labels:
```

Comment:
Since this is our chart I am ok with these defaults, BUT these really belong in the addon right?

Reply:
I did this here because it's only meant to be a subchart of kommander (hence `kommander-thanos` and also defined within that chart), but also I wanted to tie the thanos deployed in this chart specifically to this chart (and not to kommander, which the kubecost thanos will also be running under) which is why I defined the custom `app.kubernetes.io/parent-component: kommander-thanos` labels here. does that make sense?

Reply:
Yeah, like i said it makes sense in the context of our chart, so Im ok with it.